bucket/notion-enhanced.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "2.0.18-1",
+    "description": "Collaborative note taking. Notion with notion-enhancer embedded.",
+    "homepage": "https://notion-enhancer.github.io/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/notion-enhancer/notion-repackaged/releases/download/v2.0.18-1/Notion-Enhanced-2.0.18-1-win.zip",
+            "hash": "22fb539c65caf2fbbaa201ac8ce983e6bf7c4af7e05ea706302c7c4c1947165e"
+        }
+    },
+    "shortcuts": [
+        [
+            "Notion Enhanced.exe",
+            "Notion Enhanced"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/notion-enhancer/notion-repackaged",
+        "regex": "\\/releases\\/tag\\/v([\\d.]+-\\d+)"
```

```suggestion
        "regex": "/releases/tag/v([\\d.-]+)"
```
We don't need to escape `/` in Powershell. Also `[\\d.]+-\\d+` can be simplified as `[\\d.-]+`.